validator/client/registration.go

```diff
@@ -29,17 +31,31 @@ func SubmitValidatorRegistrations(
 		return nil
 	}
 
-	if _, err := validatorClient.SubmitValidatorRegistrations(ctx, &ethpb.SignedValidatorRegistrationsV1{
-		Messages: signedRegs,
-	}); err != nil {
-		if strings.Contains(err.Error(), builder.ErrNoBuilder.Error()) {
-			log.Warnln("Beacon node does not utilize a custom builder via the --http-mev-relay flag. Validator registration skipped.")
-			return nil
+	chunks := chunkSignedValidatorRegistrationV1(signedRegs, validatorRegsBatchSize)
+	var lastErr error
+
+	for _, chunk := range chunks {
+		innerSignerRegs := ethpb.SignedValidatorRegistrationsV1{
+			Messages: chunk,
+		}
+
+		if _, err := validatorClient.SubmitValidatorRegistrations(ctx, &innerSignerRegs); err != nil {
+			if strings.Contains(err.Error(), builder.ErrNoBuilder.Error()) {
+				log.Warnln("Beacon node does not utilize a custom builder via the --http-mev-relay flag. Validator registration skipped.")
```

Comment:
same issue with this log, for a large operator not running with MeV Boost (probably zero such operators these days) this log will flood.  Easy way out here is to break from the loop.